scylla-gdb.py

```diff
@@ -3789,6 +3789,49 @@ def invoke(self, arg, for_tty):
             f = reference_wrapper(f).get()
             gdb.write('%s: %s\n' % (f['_name'], f['_enabled']))
 
+class scylla_repairs(gdb.Command):
+    """ List all active repair instances for both repair masters and followers.
+
+    Example:
+
+       (repair_meta*) for masters: addr = 0x600005abf830, table = myks2.standard1, ip = 127.0.0.1, states = ['127.0.0.1->repair_state::get_sync_boundary_started', '127.0.0.3->repair_state::get_sync_boundary_finished'], repair_meta = {
+         db = @0x7fffe538c9f0,
+         _messaging = @0x7fffe538ca90,
+         _cf = @0x6000066f0000,
+
+       ....
+
+       (repair_meta*) for masters: addr = 0x60000521f830, table = myks2.standard1, ip = 127.0.0.1, states = ['127.0.0.1->repair_state::get_sync_boundary_started', '127.0.0.2->repair_state::get_sync_boundary_started'], repair_meta = {
+         _db = @0x7fffe538c9f0,
+         _messaging = @0x7fffe538ca90,
+        _cf = @0x6000066f0000,
+
+       ....
+
+      (repair_meta*) for follower: addr = 0x60000432a808, table = myks2.standard1, ip = 127.0.0.1, states = ['127.0.0.1->repair_state::get_sync_boundary_started', '127.0.0.2->repair_state::unknown'], repair_meta = {
+        db = @0x7fffe538c9f0,
+        messaging = @0x7fffe538ca90,
+        _cf = @0x6000066f0000,
```

Comment:
I think printing the entire `repair_meta` object is too much, its enough to print the address, table, ip and state, if someone wants to have a look they can print the object themselves. Something like this:
```
(repair_meta*) 0x60000432a808 kind=follower table=myks2.standard1 ip=127.0.0.1 states=[]
```
Having it all in single line helps readability a lot.

Reply:
I thought about only printing some "headers", but when we want to see other fields on all repair_meta objects, we need to loop those  repair_meta manually one by one. There can be a lot of repair_meta objects. We can redirect the output to a file and grep easily. So, I think it is better to print all details.

Reply:
We should have a flag to toggle between the two modes (summary/full). Can be a follow-up.

Reply:
Yes.